flytekit/core/tracker.py

```diff
@@ -229,16 +229,18 @@ def extract_task_module(f: Union[Callable, TrackedInstance]) -> Tuple[str, str,
         mod_name = mod.__name__
         name = f.lhs
         # We cannot get the sourcefile for an instance, so we replace it with the module
-        f = mod
+        g = mod
+        inspect_file = inspect.getfile(g)
     else:
-        mod = inspect.getmodule(f)
+        mod = inspect.getmodule(f)  # type: ignore
```

Comment:
couldn't find an easy way to fix this 
mypy error is `expression has type "Optional[Module]", variable has type Module`

I tried  `mod = inspect.getmodule(f) or None` (and also to ensure that variable is None if  `inspect.getmodule(f)` is None due to check afterwards) ....... however that didn't seem to work. Any ideas ?

Maybe im being silly but I actually don't understand why mypy is expecting mod to be  `Module`  - im guessing this is based on previous assignment to `mod = importlib.import_module(f.instantiated_in)` but should it matter if we are reassigning it anyway ?

Reply:
I see, using ignore is fine here